docs/UserGuide.adoc

```diff
@@ -12,7 +12,7 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3
+:repoURL: https://github.com/AY1920S1-CS2103-T08-3/main
```

Comment:
our repoURL should be https://github.com/AY1920S1-CS2103-T11-3/main instead of the above.

Reply:
ok i changed it in the later commits